### ISSUE TITLE
Can't delete apps

### ISSUE THREAD

**Issue by gander-code** [Chrscool8/Homebrew-Details#13]
**Describe the bug**
When attempting to remove homebrew apps a message is displayed: "Issue removing file"

**To Reproduce**
1. Open App Details
2. Navigate to Manage tab (selected by default for me)
3. Under "File Management Actions", choose "Delete App"
4. See error in the top right.

**Expected behavior**
The app should be successfully deleted.

**System Type and Version:**
 - App Version 1.05
 - SysNAND
 - OS Version 13.0.0
 - Atmosphere 1.1.1
 - Launched via R + Game

**Additional Context**
Hekate boot config:
```
[ATMOS (SYSNAND)]
emummc_force_disable=1
payload=bootloader/payloads/fusee.bin
icon=bootloader/res/icon_payload.bmp
nosigchk=1
{}
```
not sure if that matters but figured I'd include it

**Comment by CrackedDev** [comment]
Same issue here. Error "Issue Removing File"

**Comment by gander-code** [comment]
> Same issue here. Error "Issue Removing File"

Yeah still happening for me as well. Kinda makes the app pointless if I can’t manage with it.

**Comment by Niii** [comment]
Same here

**Comment by MKindy** [comment]
Same here. We're at what, now, 17 months? I'd be up for troubleshooting, but it doesn't seem the app has a debug mode or debug log.

**Comment by Chrscool8** [comment]
I have a strict minimum 28 month response time!

Jk I haven't even turned on my Switch in about this long. I'll put this on my list of things to mess around with soonish. Sorry about the inconvenience.

**Comment by gander-code** [comment]
> I have a strict minimum 28 month response time!
> 
> Jk I haven't even turned on my Switch in about this long. I'll put this on my list of things to mess around with soonish. Sorry about the inconvenience.

No worries, thanks for even getting around to responding haha. If opening this issue has brought the problem to your attention and provided a way to track its status then I'm happy.
